components/game/screen.js

```diff
@@ -252,7 +254,7 @@ class GameScreen {
     } else if (this.collisionDelay < collisionDelayCount) {
       this.collisionDelay++;
     } else {
-      this.playDropSound();
+      this.speaker.playSound(this.dropSound);
```

Comment:
Nit: I think this can be changed to without a parameter.

Please consider moving this.dropSound to the sound module.

Reply:
It was because the module should have **independence**. So, I thought audio elements must be divided from the speaker module.

Reply:
I see. It makes sense.
But the current dependency between screen and audio resources seems to be separated.
How about adding a constant class 'AudioEffects' or 'AudioResources' that contains the audio resources.

It might be used like `this.speaker.playSound(AudioResources.dropSound)`.

Reply:
It sounds great. I will change it too.